packages/dashboard/examples/dashboard-gallery.json

```diff
@@ -1,91 +1,109 @@
 {
```

Comment:
can this be deleted?

Reply:
This is one of the main testing dashboard configs, so I'm not sure we want to do that.